CONTRIBUTING.md

```diff
@@ -254,5 +254,20 @@ To enable verbose debug output declare `DEMO_DEBUG=1` like
 make demo DEMO_URL=https://failover.test.exampledns.tk DEMO_DEBUG=1
 ```
 
+## Release process
```

Comment:
@ytsarev, you might want to update the TOC at the beginning of the document.

Reply:
@somaritane updated, thanks for the catch.